src/docs/manual/02_install.adoc

```diff
@@ -78,3 +80,15 @@ The first build generated some file within the `<project>/build` folder:
 
 If you didn't get the right output, please raise an issue on https://github.com/rdmueller/docToolchain/issues[github]
 
+== Custom configuration
+
+If you want to use the same docToolchain for multiple projects, you need to create a copy of Config.gradle and ConfluenceConfig.gradle for each project.
+The paths to those configuration files can be provided by giving -P option to gradlew, for example:
+
+[source, bash]
+----
+./gradlew -PconfluenceConfigFile=$HOME/projectA/ConfluenceConfig.gradle -PmainConfigFile=$HOME/projectA/Config.gradle
```

Comment:
hm. I guess these should be .groovy files, shouldn't they?
and shouldn't we name it 'config' starting with a lower case?

Reply:
I created a new pull request fixing mistake in file names, they should be gradle of course.
If they should be upper or lower case, I don't have opinion, just existing ConfluenceConfig.groovy was upper case.

Reply:
ah - yeah. I guess I should also rename this to confluenceConfig....